src/libcollections/string.rs

```diff
@@ -1015,8 +1015,7 @@ impl String {
     }
 
     /// Create a draining iterator that removes the specified range in the string
-    /// and yields the removed chars from start to end. The element range is
-    /// removed even if the iterator is not consumed until the end.
```

Comment:
ok, I mean that:

```rust
let mut v = vec![1, 2, 3];
v.drain(..).take(1).collect::<Vec<_>>();
```

All elements in the range are removed even if  only 1 element is taken ("extracted") from the iterator.